integrations/jenkins/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,33 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: job
+    selector:
+      query:  "true"
+    port:
+      entity:
+        mappings:
+          identifier: 'if .url | contains("://") then (.url | split("://")[1] | sub("^.*?/"; "")) else .url end | sub("%20"; "-"; "g") | sub("/"; "-"; "g") | .[:-1]'
```

Comment:
is there any other way to get the id for the job & build?

Reply:
The build and job objects both contain the URL as their primary identifier. This ensures the integrity of the mapping between builds and jobs, as we can always rely on the URLs to maintain a consistent reference. For example, a job named "Project Name" with builds 34 and 35 would have the following URLs:

Job: `https://jenkin_host/job/Project%20Name`
Builds:

- `https://jenkin_host/job/Project%20Name/34`
- `https://jenkin_host/job/Project%20Name/35`

While using other fields like name or displayName as identifiers could be an option, it could complicate the process of determining the job ID from a build URL. We would need to parse the URL and extract the relevant information to match it with the job ID we expect.